src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/AlertAddressForm.tsx

```diff
@@ -99,9 +100,16 @@ export const AlertAddressForm = (props: Props) => {
     },
   });
 
+  const recordTelemetry = useTelemetry();
+
   const handleMonthlyMonitorReportToggle = () => {
     const newValue = !activateMonthlyMonitorReport;
     setActivateMonthlyMonitorReport(newValue);
+    recordTelemetry("button", "click", {
+      // There is already a unit test written to check if the right telemetry event fires when a user opts out
+      /* c8 ignore next */
```

Comment:
Let’s try to make adding `c8` ignore comments an exception. The reason for the previous coverage test failure is that only `monthly_report_opt_out` is not covered with the added unit test. We can add another test case that checks the code branch for `newValue === true` and remove this comment.